packages/send-and-read-emails/backend/java/src/main/java/Server.java

```diff
@@ -0,0 +1,242 @@
+			halt(401, "Unauthorized");
+			return null;
+		}
+
+		User user = MockDB.findUser(auth);
+		if(user == null) {
+			halt(401, "Unauthorized");
+		}
+
+		return user;
+	}
+
+	/**
+	 * Additional routes for the use case example
+	 */
+	private static void routes() {
+		post("/nylas/send-email", (request, response) -> {
+			User user = isAuthenticated(request);
+			Map<String, String> requestBody = new Gson().fromJson(request.body(), JSON_MAP);
+
+			// Create a Nylas API client instance using the user's access token
+			NylasAccount nylas = new NylasClient().account(user.getAccessToken());
+
+			// Create a new draft object
+			Draft draft = new Draft();
+
+			// Fill draft with the contents from the payload
+			draft.setFrom(new NameEmail(null, user.getEmailAddress()));
+			draft.setTo(Collections.singletonList(new NameEmail(null, requestBody.get("to"))));
+			draft.setSubject(requestBody.get("subject"));
+			draft.setBody(requestBody.get("body"));
+
+			// Send the draft
+			Message message = nylas.drafts().send(draft);
+
+			// Return the sent message object
+			return GSON.toJson(message);
+		});
+
+		get("/nylas/read-emails", (request, response) -> {
+			User user = isAuthenticated(request);
+
+			// Create a Nylas API client instance using the user's access token
+			NylasAccount nylas = new NylasClient().account(user.getAccessToken());
+
+			/*
+			 * Retrieve the first 5 threads from the Nylas API
+			 * chaining expanded gives us the full thread object
+			 */
+			RemoteCollection<Thread> threads = nylas.threads().expanded(new ThreadQuery().limit(5));
+			ArrayList<String> threadList = new ArrayList<>();
+
+			threads.forEach(thread -> threadList.add(GSON.toJson(thread)));
+			return threadList;
+		});
+
+		get("/nylas/message", (request, response) -> {
+			User user = isAuthenticated(request);
+
+			// Create a Nylas API client instance using the user's access token
+			NylasAccount nylas = new NylasClient().account(user.getAccessToken());
+
+			String messageId = request.queryParams("id");
+
+			return GSON.toJson(nylas.messages().get(messageId));
+		});
+
+		get("/nylas/file", (request, response) -> {
+			User user = isAuthenticated(request);
+
+			// Create a Nylas API client instance using the user's access token
+			NylasAccount nylas = new NylasClient().account(user.getAccessToken());
+
+			// Fetch the file from the Nylas API
+			String fileId = request.queryParams("id");
+			File file = nylas.files().get(fileId);
+
+			// Set the file's content type as the response type
+			response.type(file.getContentType());
+
+			// Download the file
+			return nylas.files().downloadBytes(fileId);
+		});
+	}
+
+	/**
+	 * Loads .env file. Tries local directory first then a few directories up.
+	 * @return The .env file contents
+	 */
+	private static Dotenv loadEnv() {
+		try {
+			return Dotenv.configure().load();
+		} catch (DotenvException e) {
+			return Dotenv.configure()
+					.directory("../../../../")
+					.load();
```

Comment:
Is there a better way?

Reply:
@AaronDDM For the `use-cases` repo, we have a top-level .env file for all apps. So technically that env file is located beyond the use case.